qiskit/optimization/exceptions/__init__.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""The exceptions module for the Qiskit Optimization stack."""
+
+from .qiskit_optimization_error import QiskitOptimizationError
```

Comment:
Do we need a whole package for this? Normally the exception just sits in the domain folder root. Terra bundles more than one exception into a single file there is that were a concern but I just see the one file/exception.

Reply:
They don't always, for instance the CircuitError, DAGCircuitError, ExtensionsError, PulseError all have their own module. To have this done the same way I put this in a separate file itself. I think we should do this in the rest of Aqua, too, so it is consistent all over Qiskit.